libs/platform/core/src/services/app-initializer/app-initializer.ts

```diff
@@ -0,0 +1,19 @@
+import { Observable } from 'rxjs';
+
+export const AppInitializerService = 'oryx.AppInitializerService';
+export const AppInitializer = 'oryx.AppInitializer*';
+
+export interface AppInitializerService {
+  initialize(): Promise<void>;
+}
+
+export interface AppInitializer {
```

Comment:
would be nice to be able to use just promise or observable, not full blown class